drivers/syslog/vsyslog.c

```diff
@@ -215,7 +215,7 @@ int nx_vsyslog(int priority, FAR const IPTR char *fmt, FAR va_list *ap)
 #endif
 
 #if defined(CONFIG_SMP)
-                             , up_cpu_index()
```

Comment:
should be up_cpu_index() not this_cpu()

Reply:
In SMP, up_cpu_index()/this_cpu() are the same here

Reply:
Yes, it correct to keep `up_cpu_index()`,  just make `this_cpu()` is arch independent as possible.

Reply:
let us keep up_cpu_index(), I will use this ability in the future